app/controllers/admin/product_import_controller.rb

```diff
@@ -2,6 +2,7 @@
 
 require 'roo'
 
+# rubocop:disable Metrics/ClassLength
```

Comment:
We use these statements for things we don't intend to fix. But if this class is too long then it should be listed in the `.rubocop_todo.yml` file and fixed at some point.